app/helpers/view_components/site_header.rb

```diff
@@ -23,8 +23,8 @@ def announcement_bar
         tag.div(class: "lg-container") do
           graphical_icon(:premium, css_class: 'h-[16px] w-[16px] mr-12') +
             tag.span("Exercism Premium has launched.") +
-            tag.strong("Supercharge your experience!") +
-            tag.div("Check it out", class: 'btn-primary btn-xs ml-4')
+            tag.strong("Click here to learn more.") +
+            tag.div("Check it out", class: 'btn-primary btn-xs ml-4 hidden md:block')
```

Comment:
Why have we lost "Supercharge your experience"?

Reply:
Sorry, this is a wrong version. It got lost in branch juggling. :(